commit 86196d88c1c4512c915e1e0ec489c10d2001d31e
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-23T12:11:01-07:00 (3 months ago)
    
    TRITON-1828 drop firmware-tools and "headnode-joyent" flavour headnode build (change jr metadata for sdc-headnode to have the build component name be 'headnode')

diff --git a/tools/jr-manifest.json b/tools/jr-manifest.json
index 67fdd26..6ee8a81 100644
--- a/tools/jr-manifest.json
+++ b/tools/jr-manifest.json
@@ -646,8 +646,7 @@
             "name": "sdc-headnode",
             "labels": {
                 "release": true,
-                "mg": "headnode-joyent",
-                "buildisprivate": true
+                "mg": "headnode"
             }
         },
         {
